modules/33_CDR/DAC/equations.gms

```diff
@@ -23,7 +32,7 @@ q33_capconst_dac(t,regi)..
 	v33_emiDAC(t,regi)
 	=e=
 	- sum(teNoTransform2rlf_dyn33(te,rlf2), vm_capFac(t,regi,"dac") * vm_cap(t,regi,"dac",rlf2))
-	-  (1 / pm_eta_conv(t,regi,"gash2c")) * fm_dataemiglob("pegas","seh2","gash2c","cco2") * vm_otherFEdemand(t,regi,"fegas")
+	-  (1 / pm_eta_conv(t,regi,"gash2c")) * fm_dataemiglob("pegas","seh2","gash2c","cco2") * v33_FEdemand(t,regi,"fegas", "fehes", "dac")
```

Comment:
would be more transparent to have 2 variables here: one for the CO2 captured from the atmosphere, and one for the CO2 captured from burning (fossil) gas. Both need to be subtracted from total emissions in the core, because the emissions from gas burning are already accounted for at the PE2SE level, but only the first is CDR (when stored), while the second should be reported as (fossil) CCS. But what if the gas comes from biomass? Then it would be CDR, too!